src/handbook/design/process.md

```diff
@@ -16,7 +18,19 @@ been conducted on the high-level UX, and development is not blocked at the start
 of a release sprint. Both visual and UX design iterations will occur throughout the
 first two weeks of a cycle, alongside development efforts.
 
-### Delivery of Art
+### Website and rest of design assets
+
+In contrast to the platform, changes and updates to the web design can be implemented independently and are not limited to specific release cycles. The timing and order of these changes, as well as the creation of other design assets such as blog images, open graphs, presentations, illustrations, etc., depend on prioritization aligned with the company's interests.
+
+## Reviewing
+
+During this phase, it is essential, whenever possible, to thoroughly review the proposed content before proceeding with the implementation in code, particularly regarding UX/UI design.
+
+This phase aims to ensure that the design not only looks visually appealing but also effectively communicates the intended message to the audience. This includes assessing the organization of information, evaluating the readability and coherence of text, and verifying the alignment with the project's objectives.
+
+By including a review phase in the design process, we can identify any potential issues or areas for improvement early on. This proactive approach helps streamline the overall design workflow, minimizes the need for extensive revisions at later stages, and ultimately delivers a high-quality design that meets both aesthetic and content-related requirements.
```

Comment:
Iteration is easier/faster in Figma, no doubt about that, and review/discussion there should be encouraged, but to block development on _something_ even as a first "this is okay" pass, feels too strict an approach, and could slow down iteration/evolution of the site.

Reply:
Ok, I see your point, this comes from a couple of times that I created a redesign for a page, and in the design phase some important things things were missed, related to content, and then asked to be changed to approve the pr.

So, how do you think we could prevent this from happening at a final stage without being too strict? Because it still blocked the iteration...

Seems like the design is "distracting" and the reviewers only read through when it has been developed.

Reply:
> this comes from a couple of times that I created a redesign for a page, and in the design review phase some important things things were missed,

That's not acceptable, and absolutely understand where you're coming from for this.

> Seems like the design is "distracting" and the reviewers only read through when it has been developed.

Not an attitude we should have at FlowForge, so I understand.

Reply:
For the balance to be effectively achieved, I believe our role in Design is to offer enough fidelity that it warrants opinions and conversation, but it doesn't have to be a fully fledged 100% accurate reflection of the final product. Iteration, at the design _or_ development phase is fine.

For example, just modifying some of the written content in the development phase is very easy, no need to have that 100% defined at the design phase.

Reply:
How about adding a piece of text in here along the lines of:

```
Design review is concluded with sign off of Figma designs from the page owner, this is more often than not the DRI for the Website, but may be Product Manager in an example like the Product Roadmap.

This does not mean iterations cannot be made in the development phase on content and layout, but attempts should be made where possible to minimise these as iterations within the design phase are for quicker and overall, save time.
```

Reply:
Ok @joepavitt,I like your proposal, and perhaps we can do without the first paragraph. How about this revised version?

## Design Review

This phase aims to ensure that the design not only looks visually appealing but also effectively communicates the intended message to the audience. This includes assessing the organization of information, evaluating the readability and coherence of text, and verifying the alignment with the project's objectives.

By including a review phase in the design process, we can identify any potential issues or areas for improvement early on. This proactive approach helps streamline the overall design workflow, minimises the need for extensive revisions at later stages, and ultimately delivers a high-quality design that meets both aesthetic and content-related requirements.

The design review concludes with the sign-off of Figma designs by the page owner, who is usually the DRI for the Website but may be another team member.

This does not mean that iterations cannot be made during the development phase on content and layout. However, it is advisable to make efforts to minimise these iterations whenever possible, as iterations within the design phase are generally quicker and ultimately save time.

Reply:
That's good and I think we could delete the first paragraph of this phase.

How about this revised version? I also made reference to the review of the Art Requests.

## Design Review

This phase aims to ensure that the designs not only look visually appealing, but also effectively communicate the intended message to the audience.

To achieve this, please assess the organization of information, evaluate the readability and coherence of text, and verify the alignment with the project's objectives.

For UX/UI proposals, the design review concludes with the sign-off of Figma designs by the page owner, who is usually the DRI for the Website but may be another team member. As for the rest of the design assets, it's determined by the person who created the Art Request.

While iterations can be made during the development phase for UX/UI design in terms of content and layout, it is recommended to minimise them whenever possible. This approach helps streamline the design workflow, allowing for quicker iterations within the design phase and ultimately saving time.

Reply:
> While iterations can be made during the development phase for UX/UI design in terms of content and layout, it is recommended to minimise them whenever possible.

Reads as if we want to minimise development iterations generally, which is not the case. We want to "fail fast", get feedback in the design phase, such that (hopefully) the development phase is more linear as there is already an agreed upon target.

Reply:
Ok, how about we replace the last paragraph with:

While iterations can be made during the development phase for UX/UI design in terms of content and layout, it is recommended to facilitate feedback loops during the design phase. By getting early feedback, we aim to identify and address any potential issues before entering the development phase. This approach helps streamline the design workflow, allowing for quicker iterations and ultimately saving time.

Reply:
that reads well

Reply:
@joepavitt I just commited the updated version